vmware-base/src/main/java/com/cloud/hypervisor/vmware/mo/HypervisorHostHelper.java

```diff
@@ -956,6 +952,11 @@ public static boolean isSpecMatch(DVPortgroupConfigInfo currentDvPortgroupInfo,
                 specMatches = false;
             }
         }
+        if (currentPortSetting.getMacManagementPolicy() != null && newPortSetting.getMacManagementPolicy() != null &&
+                currentPortSetting.getMacManagementPolicy().getMacLearningPolicy() != null && newPortSetting.getMacManagementPolicy().getMacLearningPolicy() != null &&
+                currentPortSetting.getMacManagementPolicy().getMacLearningPolicy().isEnabled() != newPortSetting.getMacManagementPolicy().getMacLearningPolicy().isEnabled()) {
```

Comment:
Can you extract this big `if` clause into a method to improve readability? Then this could be like:
````
if (hasMacLearningPolicyChanged(currentPortSetting, newPortSetting) {
   specMatches = false;
}
````